sql/core/src/test/resources/sql-tests/results/postgreSQL/timestamp.sql.out

```diff
@@ -242,11 +242,11 @@ SELECT '' AS `54`, d1 - timestamp '1997-01-02' AS diff
 -- !query 26 schema
 struct<54:string,diff:interval>
 -- !query 26 output
-	interval -236720 hours
-	interval 0 microseconds
-	interval 3 hours 4 minutes 5 seconds
-	interval 41393 hours 19 minutes 20 seconds
-	interval 953 hours 32 minutes 1 seconds
+	-236720 hours
+	0 seconds
+	3 hours 4 minutes 5 seconds
+	41393 hours 19 minutes 20 seconds
+	953 hours 32 minutes 1 seconds
```

Comment:
This diff is related to the recent change: https://github.com/apache/spark/pull/26401

Reply:
Thank you for updating, @maropu .